robotino_bringup/robots/uibk-robotino2-sh/urdf/robotino.urdf.xacro

```diff
@@ -6,11 +6,12 @@
   <xacro:include filename="$(find robotino_description)/urdf/base.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/hokuyo.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/kinect.urdf.xacro" />
+  <xacro:include filename="$(find robotino_description)/urdf/base_shell.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/pan_tilt_axis.urdf.xacro" />
   <xacro:include filename="$(find robotino_description)/urdf/arm.urdf.xacro" />
   <xacro:include filename="$(find softhand_description)/model/softhand_withCableCanal.urdf.xacro" />
 
-  <xacro:base name="base" has_8dof="false"/>
+  <xacro:base name="base" has_8dof="true"/>
```

Comment:
ok, seems I'm missing sth; what is this flag for? I assumed we need it for planning for both the arm and the base?